tasks/publish.js

```diff
@@ -48,7 +33,6 @@ const publishRelease = ({ notes, tag, token }) =>
   })
 
 async function main() {
-  build(config.default_network)
```

Comment:
This seems like a strange change in this PR

Reply:
Actually it was untested code left over from a sloppy merge.  The ESLint error caught the bug!